### PR TITLE
added changes for jira-0001

### PR DESCRIPTION
**Pull request by mohanrk051**
changes second time project-jira-0001

### CODE REVIEW

File: app.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import java.util.*;
```
Comment (Mohankempe) [inline app.java:1]: good work

**Review by Mohankempe — COMMENTED** [review]
good complete python also

File: app.java (hunk 1 above)
Comment (mohanrk051) [reply app.java:1]: welcome

**Comment by mohanrk051** [comment]
> good complete python also

ok..mithun

**Comment by Mohankempe** [comment]
merged

**Pull request merged**